src/common/auth/signup.tsx

```diff
@@ -80,7 +90,6 @@ export default function Signup({ onLogin, onClose }: Props) {
           onError={() => {
             console.log("Login Failed");
           }}
-          useOneTap

```

Comment:
Because useOneTap will show login with google popup on top left.
But right now that popup is coming when we open login/signup button. But that should ideally open when we open the site for the first time.